code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -339,6 +350,7 @@
 	moduleselect_icon = "service"
 	special_light_key = "service"
 	feedback_key = "cyborg_service"
+	hat_offset = INFINITY // No hats

```

Comment:
Set this to 0, and in the `be_transformed_to()` for this module set it to INFINITY for Tophat and 3 for Kent.